theano/tensor/nnet/abstract_conv.py

```diff
@@ -1613,7 +1612,7 @@ def perform(self, node, inp, out_):
         if isinstance(mode, tuple):
             pad = tuple(int(mode[i]) for i in range(self.convdim))
             mode = "valid"
-            new_img = numpy.zeros((img.shape[0], img.shape[1]) +
+            new_img = np.zeros((img.shape[0], img.shape[1]) +
                                   tuple(img.shape[i + 2] + 2 * pad[i]
```

Comment:
Ahm, you should do a local clone of the repository. Then checkout your `patch-1` branch (`git checkout branch-1`), modify the file, do `git add` with the file name, then `git commit --amend --no-edit` to amend the previous commit, and `git push --force` to update this PR with the new commit. And close your second PR #5492 to avoid confusion.